src/utils/browser.ts

```diff
@@ -39,3 +39,14 @@ export function getStytchJwt() {
 
 export const openTab = (url: any) =>
   featheryWindow().open(url, '_blank', 'noopener');
+
+// maps url query params to an object, used to set hidden fields
+export const mapUrlParamsToHiddenFields = () => {
```

Comment:
can this function just be unified into the single place where it's used in saveUrlParams?

Reply:
also shoudl hosted forms be updated to use this functionality instead?